api/V1/recordingUtil.ts

```diff
@@ -63,6 +61,45 @@ export interface RecordingQuery {
   filterOptions: null | any;
 }
 
+// How close is a station allowed to be to another station?
+export const MAX_STATION_DISTANCE_THRESHOLD_METERS = 60;
+// The radius of the station is half the max distance between stations: any recording inside the radius can
```

Comment:
Yes but we aren't using this assumption are we?  We are currently doing a less then and then a sort.